includes/wikia/tests/GlobalTitleTest.php

```diff
@@ -125,6 +125,31 @@ function testUrlsWithUtf8() {
 	}
 
 	/**
+	 * @dataProvider mainPageDataProvider
+	 */
+	function testMainPage( $mediaWikiMainpageContent, $exists, $expectedUrl ) {
```

Comment:
could you rename func name or comment about it to explain what is the subject of these tests? (i assume it's about title or URL)